Singularity.nemo

```diff
@@ -255,7 +267,7 @@ From: ubuntu:20.04
     cd $WORK_DIR
     
     mkdir /opt/nemo
-    mv /nemo/nemo/cfgs/NEMO/EXP00/nemo /opt/nemo/
+    c
```

Comment:
not sure what `c` is for

Reply:
I'm sure I fixed that in a later commit!

Reply:
In this commit: https://github.com/NOC-MSM/CoNES/pull/28/commits/9bd8a73a7fcd8a43f50a1abc4a6cc83f1394c47d